src/test/ui/imports/auxiliary/unnamed_pub_trait_source.rs

```diff
@@ -0,0 +1,10 @@
+pub struct S;
```

Comment:
Is there supposed to be a difference between this file `unnamed_pub_trait_source_overlap`? (Does order of the `mod`s matter?)